packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -19,16 +19,25 @@ import type {
 	ViewDocumentFragment,
 	DowncastWriter,
 	Model,
-	Position
+	Position,
+	ViewContainerElement
 } from 'ckeditor5/src/engine';
 import { Plugin, type Editor } from 'ckeditor5/src/core';
 import { findOptimalInsertionRange, isWidget, toWidget } from 'ckeditor5/src/widget';
 import { determineImageTypeForInsertionAtSelection } from './image/utils';
+import { DomEmitterMixin, type DomEmitter, global } from 'ckeditor5/src/utils';
+
+const IMAGE_WIDGETS_CLASSES_MATCH_REGEXP = /(image|image-inline)/;
```

Comment:
`image-inline` in this pattern is unnecessary, because `image` will match this as well. And other unwanted classes like `x-image-abc`.
```suggestion
const IMAGE_WIDGETS_CLASSES_MATCH_REGEXP = /^(image|image-inline)$/;
```
I'm not 100% sure about my suggestion, it would require some tests.

Reply:
I did some tests and you were right, thanks! 👍